Userland/Applications/Browser/Tab.cpp

```diff
@@ -374,6 +374,11 @@ Tab::Tab(BrowserWindow& window)
             on_set_cookie(url, cookie, source);
     };
 
+    view().on_update_cookie = [this](auto& url, auto& cookie) {
+        if (on_update_cookie)
+            on_update_cookie(url, cookie);
+    };
```

Comment:
Was this supposed to go into the previous commit?

Reply:
Nah, this follows the way the other commits work too, where the "Add IPC" touches everything from WebContentClient to OOPWV. Then moving the WebDriver API touches the Browser. This one was just a little different because most of the Browser side already existed.